mobile/lib/ui/actions/collection/collection_sharing_actions.dart

```diff
@@ -333,6 +333,91 @@ class CollectionActions {
     }
   }
 
+  Future<bool> deleteMultipleCollectionSheet(
+    BuildContext bContext,
+    List<Collection> collections,
+  ) async {
+    final textTheme = getEnteTextTheme(bContext);
+    final actionResult = await showActionSheet(
+      context: bContext,
+      buttons: [
+        ButtonWidget(
+          labelText: S.of(bContext).keepPhotos,
+          buttonType: ButtonType.neutral,
+          buttonSize: ButtonSize.large,
+          buttonAction: ButtonAction.first,
+          shouldStickToDarkTheme: true,
+          isInAlert: true,
+          onTap: () async {
+            for (final collection in collections) {
+              try {
+                await trashCollectionKeepingPhotos(collection, bContext);
+              } catch (e, s) {
+                logger.severe(
+                  "Failed to keep photos & delete collection",
+                  e,
+                  s,
+                );
+                rethrow;
+              }
+            }
+          },
+        ),
+        ButtonWidget(
+          labelText: S.of(bContext).deletePhotos,
+          buttonType: ButtonType.critical,
+          buttonSize: ButtonSize.large,
+          buttonAction: ButtonAction.second,
+          shouldStickToDarkTheme: true,
+          isInAlert: true,
+          onTap: () async {
+            for (final collection in collections) {
+              try {
+                await collectionsService.trashNonEmptyCollection(collection);
+              } catch (e) {
+                logger.severe("Failed to delete collection", e);
+                rethrow;
+              }
+            }
+          },
+        ),
+        ButtonWidget(
+          labelText: S.of(bContext).cancel,
+          buttonType: ButtonType.secondary,
+          buttonSize: ButtonSize.large,
+          buttonAction: ButtonAction.third,
+          shouldStickToDarkTheme: true,
+          isInAlert: true,
+        ),
+      ],
+      bodyWidget: StyledText(
+        text:
+            "Also delete the photos (and videos) present in these ${collections.length} albums from <bold>all</bold> other albums they are part of?",
```

Comment:
Need to extract this string for translation. Not sure how to do it with bold tags in it.